helm/efk-stack-app/charts/opendistro-es/templates/_helpers.tpl

```diff
@@ -56,8 +56,8 @@ We cannot have the chart label here as it will prevent upgrades.
 */}}
 {{- define "opendistro-es.labels.selector" -}}
 app: {{ template "opendistro-es.fullname" . }}
-# release: "{{ .Release.Name }}"
-# heritage: "{{ .Release.Service }}"
+release: "{{ .Release.Name }}"
+heritage: "{{ .Release.Service }}"
```

Comment:
Was this maybe commented out for a reason?

Reply:
This was a change from upstream. I guess they want a stricter match in case other instances are running in parallel.